examples/applications/wikipedia_principal_eigenvector.py

```diff
@@ -127,7 +127,7 @@ def get_redirects(redirects_filename):
 
 
 # disabling joblib as the pickling of large dicts seems much too slow
-#@memory.cache
+# @memory.cache
```

Comment:
We've decided not to fix existing flake8 errors.

Reply:
Ok sorry. What the reason behind that ?